web/src/hooks.client.ts

```diff
@@ -3,10 +3,13 @@ import type { HandleClientError } from '@sveltejs/kit';
 
 const DEFAULT_MESSAGE = 'Hmm, not sure about that. Check the logs or open a ticket?';
 
-const parseError = (error: unknown) => {
+const parseError = (error: unknown, statusCode: number) => {
   const httpError = isHttpError(error) ? error : undefined;
-  const statusCode = httpError?.status || httpError?.data?.statusCode || 500;
-  const message = httpError?.data?.message || (httpError?.data && String(httpError.data)) || httpError?.message;
+
+  const message =
+    statusCode === 404
+      ? 'This page does not exist'
+      : undefined || httpError?.data?.message || (httpError?.data && String(httpError.data)) || httpError?.message;
```

Comment:
I would prefer splitting this up into two variables like it was before. This is quite long right now

Reply:
Is it even possible to have different error messages with the current implementation ?

It's maybe easier to set ourselves different error messages with a switch case

Reply:
If the api server is down or unreachable it shows a different error.